java/src/org/certificatetransparency/ctlog/LogEntry.java

```diff
@@ -0,0 +1,12 @@
+package org.certificatetransparency.ctlog;
+
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class LogEntry {
+  public Ct.X509ChainEntry x509Entry;
```

Comment:
Yes, much POJOing. I'm momentarily stuck with `SignedCertificateTimestamp` and `DigitallySigned`, but I should be able to sort it out at some point...